lib/api/objectGetACL.js

```diff
@@ -154,7 +165,7 @@ export default function objectGetACL(authInfo, request, log, callback) {
                 if (err) {
                     log.trace('error processing request',
                         { method: 'objectGetACL', error: err });
-                    return callback(err, null, corsHeaders);
+                    return callback(err, bucket);
```

Comment:
This calls the main callback directly, shouldn't this be calling next with appropriate callback params i.e.`next(err, bucket, xml, resVersionId)`